front/src/utils/uiCoreHelpers.ts

```diff
@@ -0,0 +1,23 @@
+type SelectFixedOption =
+  | {
+      label: string;
+      id: string;
+    }
+  | undefined;
+
+// Utility to create standardized options for <Select> components.
+// `getOptionLabel`: Displays the option or a placeholder if empty.
+// `getOptionValue`: Uses the option itself as the value.
+export const createStringSelectOptions = (opts: string[]) => ({
+  options: opts,
+  getOptionLabel: (option: string) => option,
+  getOptionValue: (option: string) => option,
+});
+
+// Utility function for handling fixed label-value object arrays
+// structure may vary, and custom label/value mappings are needed.
+export const createFixedSelectOptions = (opts: SelectFixedOption[]) => ({
+  options: opts,
+  getOptionLabel: (option: SelectFixedOption) => option?.label ?? '',
+  getOptionValue: (option: SelectFixedOption) => option?.id ?? '',
+});
```

Comment:
It would probably be best to have these helpers in osrd-ui?

Reply:
Sure!

Reply:
ok, here the [issue](https://github.com/OpenRailAssociation/osrd-ui/issues/555)